std/process.d

```diff
@@ -2649,6 +2661,18 @@ class ProcessException : Exception
                                    : customMsg ~ " (" ~ lastMsg ~ ')';
         return new ProcessException(msg, file, line);
     }
+    // ditto
+    version (Windows)
+    static ProcessException newCreateProcessError(SpawnContext context,
+                                                  string file = __FILE__,
+                                                  size_t line = __LINE__)
+    {
+        if (context.arg0)
+            return newFromLastError("Failed to spawn process '" ~ cast(string)context.arg0 ~ "'");
+        if (context.shell)
+            return newFromLastError("Failed to run shell command '" ~ cast(string)context.shell ~ "'");
```

Comment:
This logic should be at the call site, not here. Either preformat the string there, or pass a `scope` delegate which does so on demand (which is less wasteful).

Reply:
done